web/components/registrationModal.tsx

```diff
@@ -0,0 +1,98 @@
+import { LoadingButton } from '@mui/lab';
+import { Box, Modal, TextField, Typography } from '@mui/material';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+import useLotteryRegister from '../hooks/useLotteryRegister';
+
+const registerSchema = Yup.object({
+  name: Yup.string().min(4).required(),
+});
+
+interface Props {
+  open: boolean;
+  onClose: () => void;
+  onSubmit: () => void;
+  selectedLotteries: Array<string>;
+}
+
+export default function RegisterModal({
```

Comment:
While naming the file from which the React component is exported please follow the Pascal case naming convention: `RegisterModal.tsx`